src/Symfony/Component/Console/Tests/ApplicationTest.php

```diff
@@ -1023,19 +1015,17 @@ public function testRunWithDispatcherAddingInputOptions()
         $this->assertEquals('some test value', $extraValue);
     }
 
+    /**
+     * @group legacy
+     */
     public function testTerminalDimensions()
     {
         $application = new Application();
+        $application->setTerminalDimensions(120, 80);
```

Comment:
To guarantee BC, this test should be left unchanged (except for tagging it as legacy)